samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -179,10 +175,95 @@ private IEnumerator PipelineChanged()
 
         yield return null;
 
-        SetPipeline(PipelineTypeDropdown.options[PipelineTypeDropdown.value].text);
+        // Set the Pipeline based on what Pipeline button is clicked
+        SetPipeline(PipelineModeText);
+
+        SceneButtonOnClick();
+    }
+
+    public void PipelineButtonOnClick()
+    {
+        StartCoroutine(PipelineChanged());
+    }
+
+    // Delay pop-up notification
+    IEnumerator SlideNotification()
+    {
+        //Wait for 2 secs.
+        yield return new WaitForSeconds(2);
```

Comment:
> Could you just bake in the 2 second delay into the animation so you dont need to create and start a coroutine?

hmmm it would be a good idea to bake in the 2 sec delay.... however, for the closing sliding animation, I didn't create a separate one, rather, I just make an inverse speed of the opening sliding animation. I'm afraid If I baked the delay time in, closing it will also delay 2 sec and it won't be instant....

![image](https://github.com/Esri/arcgis-maps-sdk-unity-samples/assets/106696949/9511551f-5723-47e5-b44f-3d650354547d)

Reply:
Cool, I think that's fine then